Hmm, SUSE doesn't use it, so what's the purpose of this patch?

--- a/build/parsePreamble.c.orig	2011-05-11 14:27:32.000000000 +0000
+++ b/build/parsePreamble.c	2011-05-11 15:59:08.000000000 +0000
@@ -521,6 +521,7 @@ static struct optionalTag {
     { RPMTAG_PACKAGER,		"%{packager}" },
     { RPMTAG_DISTRIBUTION,	"%{distribution}" },
     { RPMTAG_DISTURL,		"%{disturl}" },
+    { RPMTAG_DISTTAG,		"%{disttag}" },
     { RPMTAG_BUGURL,		"%{bugurl}" },
     { -1, NULL }
 };
--- a/macros.in.orig	2011-05-11 15:38:44.000000000 +0000
+++ b/macros.in	2011-05-11 15:59:08.000000000 +0000
@@ -317,6 +317,16 @@ package or when debugging this package.\
 #
 #%bugurl
 
+#	Configurable distribution tag, same as Disttag: tag in a specfile.
+#	The tag will be used to supply reliable information to tools like
+#	rpmfind.
+#
+# Note: You should not configure with disturl (or build packages with
+# the Disttag: tag) unless you are willing to supply content in a
+# yet-to-be-determined format at the tag specified.
+#
+#%disttag
+
 #	Boolean (i.e. 1 == "yes", 0 == "no") that controls whether files
 #	marked as %doc should be installed.
 #%_excludedocs
